packages/@okta/vuepress-site/docs/guides/refresh-tokens/use-refresh-token/index.md

```diff
@@ -2,6 +2,8 @@
 title: Use a refresh token
 ---
 
+> **Note:** When using the authorization code flow with PKCE to get the refresh token, this request does not require a `authorization` header in the call below.
```

Comment:
This is not entirely accurate- If the client is configured with token_endpoint_auth_mode = none (like in PKCE) we still require that you provide the clientId through some mechanism. This can be in the Authorization header with just the client_id base64 encoded, OR it can be in the request body.